yt/frontends/enzo_p/data_structures.py

```diff
@@ -62,10 +62,19 @@ def __init__(self, id, index, block_name, filename = None):
         self._children_ids = None
         self._parent_id = -1
         self.Level = -1
+        self.particle_count = None
 
     def __repr__(self):
         return "EnzoPGrid_%04d" % self.id
 
+    def _prepare_grid(self):
+        """ Copies all the appropriate attributes from the index. """
```

Comment:
You don't need whitespaces around the actual sentence.
Reference: https://www.python.org/dev/peps/pep-0257/